frontend/src/app/modules/data/services/geo-events.service.ts

```diff
@@ -2,30 +2,33 @@ import { Injectable } from '@angular/core';
 import { BehaviorSubject, map, Observable } from 'rxjs';
 import { IGeoEvent } from '~model/interfaces/geo-event.interface';
 
+const createMockGeoEvent = (props: any) => ({
+  id: '0',
+  coordinates: {
+    latitude: 48.858093,
+    longitude: 2.294694,
+  },
+  title: 'Start of construction of the Eiffel Tower',
+  timestamp: new Date(2022, 1, 15, 20, 50),
+  ...props
+})
```

Comment:
```suggestion
const createMockGeoEvent = (props: Partial<IGeoEvent>): IGeoEvent => ({
  id: '0',
  coordinates: {
    latitude: 48.858093,
    longitude: 2.294694,
  },
  title: 'Start of construction of the Eiffel Tower',
  timestamp: new Date(2022, 1, 15, 20, 50),
  ...props,
})
```

Note: this will require you to set add `"exactOptionalPropertyTypes": true` to `tsconfig.json`.